container/Containerfile

```diff
@@ -10,11 +10,12 @@ RUN /venv/bin/pip install ovirt-imageio.tar.gz
 # Final stage
 
 FROM alpine
+ARG TICKET_PATH=ticket.json
```

Comment:
You are right, `ARG` is not needed as is not used at build time.

The example command should be `--volume test:ticket`, and the default `ARG` is not considered during runtime anyway. It was there just so that the build command didn't complain. I tested and can just remove it.